git_machete/cli.py

```diff
@@ -143,6 +143,7 @@ def create_cli_parser() -> argparse.ArgumentParser:
         add_help=False,
         parents=[common_args_parser])
     add_parser.add_argument('branch', nargs='?')
+    add_parser.add_argument('-f', '--first', action='store_true')
```

Comment:
Actually... I'd consider renaming the full form to `--as-first-child`:
1. The meaning is clearer
2. The name is more consistent with the existing `--as-root` flag